test/bdd/features/edv_e2e_api.feature

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@edv_rest
+Feature: Using EDV REST API
+
+  @create_new_vault
+  Scenario: Client creates a new data vault
+    Given EDV server is running on "http://localhost:8080"
```

Comment:
remove hardcode port see aries code

Reply:
I'm checking to see if the server is up and reachable. Similar to https://github.com/hyperledger/aries-framework-go/blob/b5f5e60b50291e719a492459d48c195856d24eec/test/bdd/agent/agent_controller_steps.go#L54

Reply:
you don't need this step because you start using docker-compose

Reply:
done